.github/workflows/validate_pr_artifact.yml

```diff
@@ -0,0 +1,16 @@
+name: Validate PR Artifact
+
+on: 
+  - pull_request
+  - workflow_call
```

Comment:
I'm the one that just mentioned adding this, but I'm realizing we were going to do this when using a reusable workflow.  

I don't think we need it now that we've moved to using an action, since we'll just be reusing that right?  Thats my mistake.